docs/web3.contract.rst

```diff
@@ -1115,7 +1115,7 @@ Event Log Object
      'event': 'Transfer',
      'logIndex': 0,
      'transactionIndex': 0,
-     'transactionHash': HexBytes('0xa23e7ef4d2692c5cf34ee99123c9c73099e9c3b68c7850f91c1cbcb91ac327e0'),
+     'transactionHash': HexBytes('0x81691802dac3c3b44725bf2f1a6e8338a4e81175327228c479f19ae1eecbf23a'),
```

Comment:
Yeah I had actually done this and thought it would just be better to track the hash... but it's not like we're actually checking that it's hashed correctly so I don't think it matters at all. I will change it to `...`. I think that's better.